### PR TITLE
MGMT-17266: Add OCP 4.16 images

### PR DESCRIPTION
**Pull request by danmanor**
Add `OCP 4.16` images

## List all the issues related to this PR

- [ ] New Feature 
- [x] Enhancement 
- [ ] Bug fix
- [ ] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [x] Automation (CI, tools, etc)
- [ ] Cloud
- [ ] Operator Managed Deployments
- [ ] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [x] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [ ] No tests needed

## Checklist

- [x] Title and description added to both, commit and PR.
- [x] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [x] This change does not require a documentation update (docstring, `docs`, README, etc)
- [ ] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- Are the title and description (in both PR and commit) meaningful and clear?
- Is there a bug required (and linked) for this change?
- Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

**Review by adriengentil — APPROVED** [review]

File: data/default_release_images.json [hunk 1]
```diff
@@ -176,5 +176,37 @@
...
+        "cpu_architecture": "arm64",
+        "cpu_architectures": [
+            "arm64"
+        ],
+        "url": "quay.io/openshift-release-dev/ocp-release:4.16.0-ec.4-aarch64",
+        "support_level": "beta",
+        "version": "4.16.0-ec.4"
+    },
+    {
+        "openshift_version": "4.16-multi",
```
Comment (gamli75) [inline data/default_release_images.json:201]: why you didn't mark it as beta?

Comment (danmanor) [reply data/default_release_images.json:201]: My bad

Comment (danmanor) [reply data/default_release_images.json:201]: Thank you

**Review by gamli75 — APPROVED** [review]

**Pull request merged**